### PR TITLE
Replace get_class() calls by ::class

### PR DESCRIPTION
**Pull request by deguif**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | 

Replace `get_class()` by `::class`

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/CoverageListener.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class CoverageListener implements TestListener
     public function __construct(callable $sutFqcnResolver = null, bool $warningOnSutNotFound = false)
     {
         $this->sutFqcnResolver = $sutFqcnResolver ?? static function (Test $test): ?string {
-            $class = \get_class($test);
+            $class = $test::class;
```
Comment (fabpot) [inline src/Symfony/Bridge/PhpUnit/CoverageListener.php]: Should be reverted as PHPUnit must still be compatible with PHP 7.1 (same for all files in this bridge).

**Review by fabpot — CHANGES_REQUESTED** [review]

**Comment by stof** [comment]
I'm really not convinced that `$object::class` is more readable than `get_class($object)`. I would be in favor of keeping the usage of `get_class`

File: src/Symfony/Bridge/PhpUnit/CoverageListener.php (hunk 1 above)
Comment (deguif) [reply src/Symfony/Bridge/PhpUnit/CoverageListener.php]: I reverted the changes on the phpunit bridge

**Comment by nicolas-grekas** [comment]
I like the change personally. It gives a consistent syntax to get the class of something. Once used to it, we won't want to get back :)

**Review by derrabus — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Review by artyuum — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @deguif.

**Pull request merged**